tests/unit/filter/test_junit2obj.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+# import json
```

Comment:
Remove if not used

Reply:
without it I am getting validation errors for older version of py `3.(6,7,8)`

Reply:
Do you mean that when you remove the line you see validation errors? Do you mind sharing how you test and the result of that test?